code/modules/projectiles/projectile/bullets.dm

```diff
@@ -34,16 +34,37 @@
 	damage = 15
 
 /obj/item/projectile/bullet/pellet/weak
+	damage = 6
+	New()
+		range = rand(8)
+
+/obj/item/projectile/bullet/pellet/weak/on_range()
+	var/datum/effect_system/spark_spread/sparks = new /datum/effect_system/spark_spread
+	sparks.set_up(1, 1, src)
+	sparks.start()
+	..()
+
+/obj/item/projectile/bullet/pellet/overload
 	damage = 3
+	New()
+		range = rand(10)
 
-/obj/item/projectile/bullet/pellet/random/New()
-	damage = rand(10)
+/obj/item/projectile/bullet/pellet/overload/on_hit(atom/target, blocked = 0)
+	..()
+	explosion(target, 0, 0, 2)

```

Comment:
Oh yeah. It wrecks windows. Walls are somewhat tough but multiple rounds should mess up regular walls.

Using it usually knocks you down for about twenty seconds. I'm alright with walls and windows getting pulverised due to the effective counter play.